java/common/src/main/java/org/apache/shindig/common/servlet/URLFilter.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please check this license header as this is inconsistent with the header of the MultiReadHttpServletRequest.java